guides/unittesting-django-wagtail.md

```diff
@@ -81,30 +81,30 @@ When writing unit tests for code in cfgov-refresh there are multiple possible te
 
 There are a few different ways to provide data for your tests to operate on.
 
-- Using [Django test fixtures](https://docs.djangoproject.com/en/1.11/topics/testing/tools/#fixture-loading) to load specific data into the database. 
+- Using [Django test fixtures](https://docs.djangoproject.com/en/1.11/topics/testing/tools/#fixture-loading) to load specific data into the database.
```

Comment:
I realize this might be slightly out of scope, but we should change the Django doc links to point to stable too.

Reply:
@willbarton: It looks like https://docs.djangoproject.com/en/stable/topics/testing/tools/#fixture-loading redirects to https://docs.djangoproject.com/en/3.0/topics/testing/tools/#fixture-loading. Since I wasn't sure when we'd be getting to 3.0, I held off on changing those links for now, but I'm happy to do that if you think the 3.0 docs are OK to point to. I was also planning on doing a similar documentation update when we get to Django 2.2, so we can always push off that decision until then, too.

Reply:
Sounds good, let's hold off on the Django doc links.